value/src/main/java/com/google/auto/value/processor/AutoValueProcessor.java

```diff
@@ -216,6 +216,7 @@ private String generatedSubclassName(TypeElement type, int depth) {
 
     private ImmutableList<String> buildAnnotations(TypeSimplifier typeSimplifier) {
       ImmutableList.Builder<String> builder = ImmutableList.builder();
+      Set<Name> methodAnnotations = null;
```

Comment:
It would probably work, but unfortunately its javadoc mentions only equals, not hashCode. For Name, the [javadoc](https://docs.oracle.com/javase/8/docs/api/javax/lang/model/element/Name.html) does mention hashCode, too